frontend/src/bundles/common/components/sidebar-nav/sidebar-nav.tsx

```diff
@@ -0,0 +1,41 @@
+import { useCallback } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+
+import { Button, ButtonSize, ButtonVariant } from '../components.js';
+
+interface SidebarNavProperties {
+    icon: JSX.Element;
+    text: string;
+    to: string;
+    isActive?: boolean;
+}
+
+const SidebarNav: React.FC<SidebarNavProperties> = ({
+    icon,
+    text,
+    to,
+    isActive = false,
+}) => {
+    const navigate = useNavigate();
+    const handleNavigation = useCallback((): void => {
+        navigate(to);
+    }, [navigate, to]);
+    return (
+        <Link
```

Comment:
We have our own Link component, please you it

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/62f0f147-9cfb-4a43-8325-8014802a9444)